.github/workflows/codecov

```diff
@@ -0,0 +1,19 @@
+name: API workflow
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Test python API
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.11'
+    - name: Install requirements
+      run: pip install -r api/requirements.txt
```

Comment:
This file doesn't exist, so what will this do?

Reply:
I've removed the useless example config and changed it to something less useless. It might not actually do much, but it will probably run at least.